apps/console/src/features/identity-providers/components/identity-provider-edit.tsx

```diff
@@ -251,6 +252,18 @@ export const EditIdentityProvider: FunctionComponent<EditIdentityProviderPropsIn
         </ResourceTab.Pane>
     );
 
+    const ConnectedAppsTabPane = (): ReactElement => (
+        <ResourceTab.Pane controlledSegmentation>
+            <ConnectedApps
+                editingIDP={ identityProvider }
+                isReadOnly={ isReadOnly }
+                isLoading={ isLoading }
+                loader={ Loader }
+                data-testid={ `${ testId }-connected-apps-settings` }
```

Comment:
Can we use `data-componentid` instead of `data-testid`?

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3494/commits/05d52641ee34bd16b1f267ba20fb05d9c9c627bf